github-pullrequest-plugin/src/main/resources/com/github/kostyasha/github/integration/branch/GitHubBranchRepository/index.groovy

```diff
@@ -18,6 +19,15 @@ def makeBuildItem(def runs) {
     }
 }
 
+def escapeBranchName(def branchName) {
+   // escape anything that isn't alphanumeric
+   return StringEscapeUtils.escapeJavaScript(branchName);
```

Comment:
groovy dep in jenkins is very questionable and differs. commons is already used in parameters creation.